edk2toolext/base_abstract_invocable.py

```diff
@@ -45,6 +45,10 @@ def GetLoggingFolderRelativeToRoot(self):
         ''' Return a path to folder for log files '''
         raise NotImplementedError()
 
+    def InputParametersConfigured(self):
```

Comment:
Maybe we should add Hook or Callback to make it a little more clear

Reply:
agree